nntrainer/include/tensor.h

```diff
@@ -51,7 +51,7 @@ class Tensor {
   /**
    * @brief     Constructor of Tensor
    */
-  Tensor() : dim(){};
+  Tensor() : _is_contiguous(true), dim(){};
```

Comment:
when `_is_contiguous` goes false?

Reply:
It will become false when non-contiguous tensor is generated (tensor that refers to the non-sequential memory).

This PR is more of a preparation for #217. For now, every tensor is contiguous. Broadcasted tensor will be non-contiguous.

Reply:
Got it